### PR TITLE
feat: adds emoji contentType support

### PR DESCRIPTION
**Pull request by richard-ramos**
Uses information from this post: https://forum.qt.io/topic/77986/emoji-support/4 and TwitterEmoji .ttf font obtained from here: https://forum.xda-developers.com/android/themes/font-twitter-emojione-emoji-replacements-t3020653  which is generated from here: https://github.com/twitter/twemoji

Worth reviewing if we'd run into any licensing issues

Screenshot from 2020-06-24 16-04-58

### CODE REVIEW

**Comment by richard-ramos** [comment]
```
proc sendChatMessage*(chatId: string, msg: string): string =
  callPrivateRPC("sendChatMessage".prefix, %* [
    {
      "chatId": chatId,
      "text": msg,
      "responseTo": nil,
      "ensName": nil,
      "sticker": nil,
      "contentType": 1
    }
  ])
```
must be updated to change the contentType to 4 if the text contains only emojis

File: ui/app/AppLayouts/Chat/ChatColumn/Message.qml [hunk 1]
```diff
@@ -19,7 +19,8 @@ Item {
     property string authorCurrentMsg: "authorCurrentMsg"
     property string authorPrevMsg: "authorPrevMsg"
 
-    property bool isMessage: contentType == Constants.messageType || contentType == Constants.stickerType || contentType == Constants.emojiType 
+    property bool isEmoji: contentType == Constants.emojiType
```
Comment (jrainville) [inline ui/app/AppLayouts/Chat/ChatColumn/Message.qml:22]:
```suggestion
    property bool isEmoji: contentType === Constants.emojiType
```

File: ui/app/AppLayouts/Chat/ChatColumn/Message.qml [hunk 2]
```diff
@@ -19,7 +19,8 @@ Item {
...
+    property bool isMessage: contentType == Constants.messageType || contentType == Constants.stickerType
```
Comment (jrainville) [inline ui/app/AppLayouts/Chat/ChatColumn/Message.qml:23]:
```suggestion
    property bool isMessage: contentType === Constants.messageType || contentType === Constants.stickerType 
```

File: ui/app/AppLayouts/Chat/ChatColumn/Message.qml [hunk 3]
```diff
@@ -236,7 +237,7 @@ Item {
 
         StyledTextEdit {
             id: chatText
-            text: message
+            text: contentType == Constants.stickerType ? "" : message
```
Comment (jrainville) [inline ui/app/AppLayouts/Chat/ChatColumn/Message.qml:240]:
```suggestion
            text: contentType === Constants.stickerType ? "" : message
```

File: ui/shared/StyledText.qml [hunk 4]
```diff
@@ -2,5 +2,5 @@ import QtQuick 2.13
 import "../imports"
 
 Text {
-    font.family: Theme.fontRegular.name
+    font.family: Theme.fontEmoji.name + "," + Theme.fontRegular.name
```
Comment (jrainville) [inline ui/shared/StyledText.qml:5]: How does it work? Doesn't it work like HTML where it uses the first font, if it doesn't find it, it uses the second one?

**Pull request merged**